### PR TITLE
ADCM-2374_1 fix tests test_activatable_groups_ui_options

### PR DESCRIPTION
**Pull request by annademina**

### CODE REVIEW

**Review by alexterent — APPROVED** [review]

File: tests/ui_tests/app/page/common/configuration/page.py [hunk 1]
```diff
@@ -92,11 +91,21 @@ def compare_versions(self, compare_with: str, base_compare_version: Optional[str
             # to hide select panel so it won't block other actions
             self.find_element(self.locators.compare_to_select).send_keys(Keys.ESCAPE)
 
-    def click_on_advanced(self):
+    @allure.step('Click on advanced button')
+    def click_on_advanced(self, is_rows_changes=False):
         """Click on advanced button and wait rows changed"""
-        with self.wait_rows_change():
+        if is_rows_changes:
```
Comment (skhomuti) [inline tests/ui_tests/app/page/common/configuration/page.py:97]: this flag is really needed? it seems unused

File: tests/ui_tests/app/page/common/configuration/page.py [hunk 2]
```diff
@@ -189,6 +198,33 @@ def _click_group():
 
         wait_until_step_succeeds(_click_group, period=1, timeout=10)
 
+    @contextmanager
+    def wait_group_changed(self, group_name: str):
```
Comment (skhomuti) [inline tests/ui_tests/app/page/common/configuration/page.py:202]: add docstring
it's strange that linter doesn't found it

File: tests/ui_tests/app/page/common/configuration/page.py [hunk 3]
```diff
@@ -243,6 +279,45 @@ def check_config_fields_visibility(
...
+    def get_items_in_group(self, group: WebElement):
+        """Get item rows in the group"""
+
+        try:
+            return self.find_children(group, self.locators.ConfigGroup.item_row, timeout=2)
+        except TimeoutException:
+            return []
+
+    @allure.step('Check that subs in group {group_name} is visible = {is_visible}')
+    def check_subs_visability(self, group_name: str, is_visible: bool = True):
```
Comment (skhomuti) [inline tests/ui_tests/app/page/common/configuration/page.py:302]: `visibility` typo

File: tests/ui_tests/test_tooltips.py [hunk 4]
```diff
@@ -1,84 +0,0 @@
-# Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (skhomuti) [inline tests/ui_tests/test_tooltips.py:1]: why these tests were removed?

**Review by skhomuti — CHANGES_REQUESTED** [review]

File: tests/ui_tests/app/page/common/configuration/page.py (hunk 1 above)
Comment (annademina) [reply tests/ui_tests/app/page/common/configuration/page.py]: yep, removed

File: tests/ui_tests/app/page/common/configuration/page.py (hunk 2 above)
Comment (annademina) [reply tests/ui_tests/app/page/common/configuration/page.py:199]: added

File: tests/ui_tests/app/page/common/configuration/page.py (hunk 3 above)
Comment (annademina) [reply tests/ui_tests/app/page/common/configuration/page.py]: changed

File: tests/ui_tests/test_tooltips.py (hunk 4 above)
Comment (annademina) [reply tests/ui_tests/test_tooltips.py:1]: i'll add it in other tests in ADCM-2387

Comment (annademina) [reply tests/ui_tests/test_tooltips.py]: returned

**Review by skhomuti — APPROVED** [review]

**Review by alexterent — APPROVED** [review]

**Pull request merged**